models/public/fastseg-large/fastseg-large.md

```diff
@@ -0,0 +1,96 @@
+# fastseg-large
+
+## Use Case and High-Level Description
+
+fastseg-large is an accurate real-time semantic segmentation model. For details see [repository](https://github.com/ekzhang/fastseg).
+
+## Specification
+
+| Metric            | Value                |
+|-------------------|----------------------|
+| Type              | Semantic segmentation|
+| GOps              | 140.9611             |
+| MParams           | 3.2                  |
+| Source framework  | PyTorch\*            |
+
+## Accuracy
+
+| Metric | Value |
+| ------ | ----- |
+| mean_iou  | 72.67%|
+
+## Input
+
+### Original model
+
+Image, name: `input0`, shape: [1, 3, 1024, 2048], format: [B, C, H, W],
+   where:
+
+    - B - batch size
+    - C - number of channels
+    - H - image height
+    - W - image width
+
+   Expected color order: RGB.
+Mean values: [123.675, 116.28, 103.53], scale values: [58.395, 57.12, 57.375]
+
+### Converted Model
+
+Image, name: `input0`, shape: [1, 3, 1024, 2048], format: [B, C, H, W],
+   where:
+
+    - B - batch size
+    - C - number of channels
+    - H - image height
+    - W - image width
+
+   Expected color order: BGR.
+
+## Output
+
+### Original Model
+
+Float values, which represent scores of a predicted class for each image pixel. The model was trained on Cityscapes dataset with 19 categories of objects. Name: `output0`, shape: [1, 19, 1024, 2048] in [B, N, H, W] format, where
+
+    - B - batch size
+    - N - number of classes
+    - H - image height
+    - W - image width
+
+### Converted Model
+
+Float values, which represent scores of a predicted class for each image pixel. The model was trained on Cityscapes dataset with 19 categories of objects. Name: `output0`, shape: [1, 19, 1024, 2048] in [B, N, H, W] format, where
```

Comment:
```suggestion
Float values, which represent scores of a predicted class for each image pixel. The model was trained on Cityscapes dataset with 19 categories of objects. Name: `output0`, shape: `1, 19, 1024, 2048` in `B, N, H, W` format, where
```

Reply:
and the same for inputs, according to internal discussion

Reply:
done